docs/decisions/characteristics/0008-observability.md

```diff
@@ -4,77 +4,65 @@ deciders: Kavya, Susmitha, Uma, Miguel, Shari
 
-Chosen option: "{title of option 1}", because
-{justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force {force} | … | comes out best (see below)}.
+Related to Architectural Style:
 
-<!-- This is an optional element. Feel free to remove. -->
+- Architectural styles with the following characteristics become more difficult to understand (regarding flow), to analyse, to debug and support. If a style is selected that falls into one of these categories, then good observability with cohesive single-pane-of-glass dashboards and good downstream automation helps to alleviate this.
+  -- Highly distributed systems
+  -- High degree of Architectural Quanta
+  -- High level of Abstractness
 
-### Consequences
+## Decision Outcome
 
-- Good, because {positive consequence, e.g., improvement of one or more desired qualities, …}
-- Bad, because {negative consequence, e.g., compromising one or more desired qualities, …}
-- … <!-- numbers of consequences can vary -->
+Because of the documented problem statement and drivers, Observability is a core characteristic of Hey Blue!
 
-<!-- This is an optional element. Feel free to remove. -->
+### Consequences
 
-## Validation
+- Platform, tool and technology selection must take ease of implementation for Observability into account (including taking into account existing probes, auto-instrumentation and the need for manual instrumentation)
+- Truly cross-functional teams must be formed so that analytics dashboards built can satisfy the needs of development, operations, support, product and management.
+- To satisfy all the needs of Observability in Hey Blue!, delivery teams must have a broad skillset including usability/user experience, performance and health testing/monitoring, product management, systems level thinking, data analytics and visualization.
 
-{describe how the implementation of/compliance with the ADR is validated. E.g., by a review or an ArchUnit test}
+## Fitness Functions
 
-<!-- This is an optional element. Feel free to remove. -->
+- For requirements driven drivers, automated functional tests tha verify logs/merics/traces are captures as expected for given use cases
+- For associated architectural characteristics drivers, capability to write and automate the required fitness functions serves as verification
+- Related to the elemeents noted under architectural styles, the trend of SRE/Incident metrics such as the following can serve as a proxy
+  -- Mean time to Detect
+  -- Mean time to Acknowledge
+  -- Mean time to Analysis (includes Triage/Isolate/Diagnose)
+- If an ideal stat of Observability drivng high value insights and downstream automation is achieved (potentially including AIOps and achieving autonomic/self-healing systems), then Mean Time to Repair/Resolve/Recover may even be considered as proxies in some scenarios
```

Comment:
I think an example of mean time as proxy(if downtime automation is achieved) might be helpful. I'm not too sure its clear

Reply:
I can't think of a good example off the top of my head.  I'll delete this point